src/main/auth/dto/changePasswordDto.ts

```diff
@@ -0,0 +1,10 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+export class ChangePasswordDto {
+  @ApiProperty()
+  password: string;
```

Comment:
https://stackoverflow.com/questions/60451337/password-confirmation-in-typescript-with-class-validator
recommend you create class or decorator confirm password